### ISSUE TITLE
Report iframe and canvas do not have a title or label

### ISSUE THREAD

**Issue by NiquiCara** [rubyforgood/abalone#535]
The iframe containing the graph, and the canvas displaying the graph, need an accessible title or label

![iframe-missing-label](https://user-images.githubusercontent.com/45127691/102561077-c1c8b280-40a1-11eb-9633-219688088f3a.PNG)

**Comment by CraigJZ** [comment]
Closed by #580
